INN/umbrella-sfpublicpress
# Update article byline area

## Changes

This pull request makes the following changes:

- Styles the article page byline area
- Copies over the `single-two-column.php` template and `content-single-classic.php` partial from Largo and modifies them so that the article header area is 100% width 

Single one column desktop:
![screencapture-sfpp-test-2020-01-15-10_13_37](https://user-images.githubusercontent.com/18353636/72445490-c5600500-377f-11ea-9709-c13db8524727.png)

Single one column mobile:
![screencapture-sfpp-test-2020-01-15-10_14_00](https://user-images.githubusercontent.com/18353636/72445512-cc871300-377f-11ea-85b5-741326125a96.png)

Single two column desktop:
![screencapture-sfpp-test-2020-01-15-10_10_47](https://user-images.githubusercontent.com/18353636/72445368-9ba6de00-377f-11ea-9be3-d6e6dc474e38.png)

Single two column mobile:
![screencapture-sfpp-test-2020-01-15-10_11_36](https://user-images.githubusercontent.com/18353636/72445391-a3668280-377f-11ea-95b4-a53c38743c45.png)

## Why



For #28

## Testing/Questions

Features that this PR affects:

- Article page one and two column templates
- Article bylines


Questions that need to be answered before merging:

- [ ] Does everything look ok? Any other template changes needed?

Steps to test this PR:

1. View a post using the single column template
2. View a post using the two column template (make sure items are in the sidebar)
